Server/Server.Core/Server.Core.Static/ArgumentParser.hpp

```diff
@@ -17,7 +17,7 @@ class ArgumentParser
 private:
     KeysValidator                  validator;
     std::map<std::string, int32_t> arguments;
```

Comment:
It is better to remove this field. It has only been used a few times in the code, so there is no need to save it.